File: RAI/Clase13/Java/Perro.java
//package RAI.Clase13.Java;

public class Perro extends Animal{
    private int fuerzaMordida; 

    public Perro(String nombre, String familia, String especie, int fuerzaMordida){
        super(nombre, familia, especie);
        this.fuerzaMordida = fuerzaMordida;
    }

    public void olfatea(){
        System.out.println(this.nombre + " esta olfateando...");
    }
    
}

File: RAI/Clase13/Python/Perro.py
from Animal import Animal

class Perro(Animal):
    def __init__(self, nombre: str, familia: str, especie: str, fuerza_mordida:int):
        super().__init__(nombre, familia, especie)
        self.fuerza_mordida = fuerza_mordida

    def olfatea(self):
        print(self.nombre + " olfatea")

    def hace_sonido(self):
        print(self.nombre + " hace Guau...")